doc/develop/optimizations/tools.rst

```diff
@@ -229,32 +229,32 @@ as in the following example.
 
 Pahole will generate something similar to the output below in the console::
 
-    /* Used at: zephyr/isr_tables.c */
-    /* <80> ../include/sw_isr_table.h:30 */
-    struct _isr_table_entry {
-            void *                     arg;                  /*     0     4 */
-            void                       (*isr)(void *);       /*     4     4 */
+    /* Used at: […]/build/zephyr/kobject_hash.c */
```

Comment:
I don't think we've quite settled on what to do with Unicode characters so in the meanwhile do you mind doing `s/…/.../g`?